### PR TITLE
fix #61823 voiranime.com

### LINKED ISSUE #61823: voiranime.com
### Issue URL (Anti Adblock Script)
 [http://voiranime.com/kimetsu-no-yaiba-demon-slayer-01-vostfr/#](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fvoiranime.com%2Fkimetsu-no-yaiba-demon-slayer-01-vostfr%2F%23)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.4.31
Browser: | Opera
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (2880), <br/>Block WebRTC, <br/>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Language-specific:</b><br/> AdGuard French, <br/>AdGuard Russian, <br/>AdGuard German, <br/>AdGuard Japanese, <br/>AdGuard Spanish/Portuguese, <br/>EasyList Italy

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#61823

### CODE REVIEW

File: EnglishFilter/sections/whitelist.txt [hunk 1]
```diff
@@ -227,6 +227,8 @@ copypastecharacter.com#$##contents { visibility: visible!important; }
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/57217
 @@||local.adguard.org^
+! https://github.com/AdguardTeam/AdguardFilters/issues/61823
+@@||cdnrole.com/videojs7/video-js-*/video.min.js$domain=gounlimited.to
```
Comment (Alex-302) [inline EnglishFilter/sections/whitelist.txt:231]: it is not blocked

File: FrenchFilter/sections/whitelist.txt [hunk 2]
```diff
@@ -6,6 +6,8 @@
 /^(https?|wss?):\/\/([0-9a-z\._-]+)\.(accountant|bid|cf|click|club|com|cricket|date|download|faith|fun|ga|gdn|gq|info|life|link|loan|men|ml|network|online|ovh|party|pro|pw|racing|review|rocks|ru|science|site|space|stream|tk|top|trade|uk|webcam|website|win|xyz|zone)\.\/(.*)/$document,image,popup,script,stylesheet,subdocument,websocket,xmlhttprequest,badfilter
 ! https://github.com/AdguardTeam/AdguardFilters/issues/61786
 willhaben.at#@#.adHeading
+! https://github.com/AdguardTeam/AdguardFilters/issues/61823
+@@||voiranime.com/wp-content/cache/minify/7a103.js
```
Comment (Alex-302) [inline FrenchFilter/sections/whitelist.txt:10]: also  not blocked

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: FrenchFilter/sections/whitelist.txt (hunk 2 above)
Comment (Sergey-Lyapin) [reply FrenchFilter/sections/whitelist.txt:10]: For fixing switching between players
Screenshot 2020-08-21 at 11 23 46

File: EnglishFilter/sections/whitelist.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply EnglishFilter/sections/whitelist.txt:231]: Now I don't see this request too and the video works good

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**